cypress/integration/articleBodySpec.js

```diff
@@ -0,0 +1,126 @@
+import config from '../support/config/services';
+import {
+  copyrightDataWindow,
+  firstHeadlineDataWindow,
+  firstParagraphDataWindow,
+  firstSubheadlineDataWindow,
+  getElement,
+  placeholderImageLoaded,
+  renderedTitle,
+  visibleImageNoCaption,
+  visibleImageWithCaption,
+  shouldContainText,
+} from '../support/bodyTestHelper';
+
+Object.keys(config)
+  .filter(service => config[service].pageTypes.articles !== undefined)
+  .forEach(service => {
+    describe(`${service} Article Body Tests`, () => {
+      // eslint-disable-next-line no-undef
+      beforeEach(() => {
+        cy.visit(
+          `/${service}/articles/${config[service].pageTypes.articles.asset}`,
+        );
+      });
+
+      it('should render an H1, which contains/displays a styled headline', () => {
+        firstHeadlineDataWindow();
+      });
+
+      it('should render a formatted timestamp', () => {
+        cy.window().then(win => {
+          if (win.SIMORGH_DATA.pageData.metadata.language === 'en-gb') {
```

Comment:
this is used 3 times so maybe do 

```js
if (isEnglishLanguage) {
```

Reply:
yeah thanks @jroebu14 will fix the above two mentioned ones.

Reply:
I have fixed the first one but not the second one as its causing me issues so would prefer to leave it as it is.